content/_releases/22/32.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Product Update - v22.32
+description: Details of the product update v22.32 from 11th of August 2022.
+layout: rel
+version: "22.32"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-08-11
+redirect_from:
+  - /releases/22.32.html
+---
+
+
+## Improvements and Updates
+
+### Continued improvements
+
+Here we should talk about the automated vulnerabilities checks of our code.
```

Comment:
As part of ongoing program of improving our code base and in-line with our ISO27001 processes we have recently completed a number of improvements to both address existing vulnerabilities and prevent potential vulnerabilities being released. Instrumental to this approach is the use of regular scans of our code base during back ups.

An high-level overview of our InfoSec standards and processes are available here (link to the high level infosec doc)

Reply:
your are the best